packages/rolldown/src/experimental-index.ts

```diff
@@ -44,6 +44,17 @@ export {
   viteWebWorkerPostPlugin,
 } from './builtin-plugin/constructors';
 
+export {
+  /**
+   * @deprecated Use `viteDynamicImportVarsPlugin` instead.
+   */
+  viteDynamicImportVarsPlugin as dynamicImportVarsPlugin,
+  /**
+   * @deprecated Use `viteImportGlobPlugin` instead.
+   */
```

Comment:
```suggestion
  /**
   * Alias of `viteDynamicImportVarsPlugin`. Note that this plugin is only intended to be used by Vite.
   */
  viteDynamicImportVarsPlugin as dynamicImportVarsPlugin,
  /**
   * Alias of `viteImportGlobPlugin`. Note that this plugin is only intended to be used by Vite.
   */
```
I think we don't have to mark it as deprecated as we'll be renaming back these. (While we won't be for other plugins)